logic_operators/logic_operators.php

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
```

Comment:
@Niki1993 
Great job with homework 3. :+1:  Please rename the folder from "logic_operators" to "Homework-3" and submit again : )